File: source/tests/DataDynamics.PageFX.QA/TestCases/PageFX/Simple/mono/VB/Mono/GetType.vb
Imports System

Module Test
	Function _Main() As Integer
        Dim s As String = GetType(String).ToString()
        If s <> "System.String" Then
            System.Console.WriteLine("#A1: wrong type returned") : Return 1
        End If
        Dim t As Type = GetType(String)
        If Not t Is s.GetType() Then
            System.Console.WriteLine("#A2: wrong type returned") : Return 1
        End If
    End Function

	Sub Main()
		_Main()
		System.Console.WriteLine("<%END%>")
	End Sub

End Module

File: tests/Simple/CSharp/Nullable/GetType.cs
using System;

internal class Test
{
    static void Foo(object obj)
    {
        Console.WriteLine(obj.GetType() == typeof(int));
    }

    static void Main()
    {
        int? v = 10;
        Foo(v);
        Console.WriteLine("<%END%>");
    }
}